dts/bindings/flash_controller/renesas,rx-flash.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2021 KT-Elektronik, Klaucke und Partner GmbH
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Renesas RX Flash region
+
+compatible: "renesas,rx-flash"
+
+include: base.yaml
```

Comment:
`include: flash-controller.yaml` ?

Reply:
I add new binding in mtd for flash node and remove redundant properties